src/dependencies.js

```diff
@@ -10,6 +10,20 @@ function getATSPIDepends (version, dependencyMap) {
   return semver.gte(version, '5.0.0-beta.1') ? [dependencyMap.atspi] : []
 }
 
+/**
+ * Determine whether DRM is a necessary dependency, given the Electron version.
+ */
+function getDRMDepends (version, dependencyMap) {
+  return semver.gte(version, '9.0.0-beta.1') ? [dependencyMap.drm] : []
```

Comment:
We need better checks in these new depends modules, particularly since we don't (yet) release -common, -debian, and -redhat at the exact same time. -common was released a day before -debian (and probably -redhat) and bugs happened due to there not being a `drm` entry in -debian: https://github.com/electron-userland/electron-installer-debian/pull/224#issuecomment-650612833

Reply:
Either that or we consider adding new dependencies as breaking changes (minor version bump < 1.0.0, major version bump thereafter).

Reply:
I think that any breaking change introduced by dependencies should also be reflected in the version numbering. Sometimes a dependency is removed from electron and that would just be a minor change, otherwise, they should be major changes.
Also, I was surprised to see that the previous release was 0.10.2 rather than 1.0.0. Since you've added typescript definitions and other fixes

Reply:
> Also, I was surprised to see that the previous release was 0.10.2 rather than 1.0.0.

There are two things that I want done for 1.0.0 which I didn't have time to do this weekend:

* Add docs & publish them (like Electron Packager)
* Remove `spawn` re-export

Reply:
I've added issues for both items.